test-plans/fetching/manifest.toml

```diff
@@ -0,0 +1,15 @@
+name = "fetching"
+
+[defaults]
+builder = "docker:generic"
+runner = "local:docker"
+
+[builders."docker:generic"]
+enabled = true
+
+[runners."local:docker"]
+enabled = true
+
+[[testcases]]
```

Comment:
What is this table for? Can we control which tests run with this?

Reply:
I was thinking last night if we might be able to access this value in the code, then we can control which tests run and we might not need to split tests. What do you think?

Reply:
If there are multiple binaries within one test plan, they are listed there.

Reply:
For each binary, we can specify different test cases and also specify parameters for those test cases.